src/Allegro.CosmosDb.Migrator/src/Allegro.CosmosDb.Migrator.Api/appsettings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app": {
+    "name": "CosmosMigrator Service API"
+  },
+  "cosmos": {
+    "connectionString": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
```

Comment:
What KEY is this? Isn't it sensitive? Do we need to commit it?

Reply:
all emulator (i checked windows installer and unix docker) provide same key, so this is emulator key for people who wants to play locally. will add some note to readme